standard/PortalStatistics.lua

```diff
@@ -1046,6 +1046,8 @@ function StatisticsPortal._getChartConfig(args, params)
 		yearBreakdown = Logic.readBool(args.yearBreakdown),
 		removeEmptyCategories = Logic.readBool(args.removeEmptyCategories),
 		removeEmptySeries = Logic.readBool(args.removeEmptySeries),
+		chartType = args.chartType or 'bar',
+		stackType = args.stackType or nil,
```

Comment:
or nil is usually not needed

Reply:
Yeah, I was testing it with a blank string but it was not doing what I wanted, so I was setting it explicitly to nil to check it functioned as intended